### DISCUSSION TITLE
Android Build - documentation

### DISCUSSION THREAD

**Discussion by rubensoleao** [nesbox/TIC-80#2223] [Q&A]
Is there any documentation on how to create an APK build for android?

**Comment by sogaiu** [comment]
Don't know about docs, but I think [the bits here](https://github.com/nesbox/TIC-80/blob/39be7a5e731ff1a48d230e5c3847d195f1117cec/.github/workflows/build.yml#L568) and [here](https://github.com/nesbox/TIC-80/tree/main/build/android) might have been sufficient hints (assuming Android dev know-how) to do something locally.

Don't recall the details though (^^;
